projects/smarti18n/cli/commandBuild.js

```diff
@@ -21,23 +21,24 @@ module.exports = class BuildCommand {
 	 */
 	run() {
 		this.verboseCommandInfo();
-		const projectSrcPath = this.getSrcPath(this.options.project);
-		this.build(projectSrcPath);
-		this.saveLocales(projectSrcPath);
+		const PROJECT_SRC_PATH = this.getSrcPath(this.options.project);
+		this.build(PROJECT_SRC_PATH);
+		this.saveLocales(PROJECT_SRC_PATH);
 	}
 
+
 	/**
 	 *
 	 *
 	 * @param {*} buildPath
 	 */
 	build(buildPath) {
 		this.verbose(`- ${buildPath}`);
-		const list = fs.readdirSync(buildPath);
-		const sep = this.separateFolders(buildPath, list);
-		sep.folders.forEach(folder => this.build(folder));
-		if (sep.i18n)
-			this.addDataFromDir(sep.i18n);
+		const LIST = fs.readdirSync(buildPath);
+		const SEP = this.separateFolders(buildPath, LIST);
+		SEP.folders.forEach(folder => this.build(folder));
+		if (SEP.i18n)
+			this.addDataFromDir(SEP.i18n);

```

Comment:
why change these to uppercase? I understand changing the src path but these I don't see why...

Reply:
I just like to keep constants as uppercase lol, But I can revert that

Reply:
yes but I usually prefer to use all uppercase to signalize class-wide or global-wide constants, not scoped `const variables`